examples/mobile/UIComponents/components/list/reorder.html

```diff
@@ -229,16 +224,14 @@ <h1>
 						<span class="ui-li-text-sub">
 							Multi sublines
 						</span>
-					</span>
+					</div>
 				</li>
-				<li class="ui-li-flex ui-li-multilines">
-					<span class="ui-li-area ui-li-area-a">
-						<span class="ui-li-text">
-							<span>
-								Primary text
-							</span>
+				<li>
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Multi primary text
 						</span>
-						<span class="ui-li-text-2">
+						<span class="ui-li-text-sub ui-li-text-value">
```

Comment:
Do you use sub text instead of 2?
If so, how do you handle cases of 3 or higher?

Reply:
I will change original usages.
We can place the text to the correct position by using the 'ui-li-text-title' and 'ui-li-text-sub'.